app/client/components/transferAccount/types.ts

```diff
@@ -2,6 +2,6 @@ export enum TransferAccountTypes {
   USER = "USER",
```

Comment:
To ur comment, "USER" should be nuked - it's "BENEFICIARY" in this case. Incoming change here:
https://github.com/teamsempo/SempoBlockchain/pull/441/files#diff-8cca8a6af685d1a884e93491c72ced3bR2